_assets/css/styles.scss

```diff
@@ -437,20 +437,20 @@ code.language-plaintext {
 }
 
 .cg-diagrams-stage {
-  background: #f1f1f1;
+  background: #f1f1f1 !important;
```

Comment:
It's been a while since CSS, but is there any reason we **have** to use `!important` for these rules? If the goal is just to make sure this CSS rule is used, we could try increasing the selector specificity here instead. I'm happy to pair on this

Reply:
The older versions of mermaid had the ability to apply a `null` theme which didn't add CSS rules. This version requires a theme to be set, so now we're setting the theme to `default`. The updated way to customize the theme is to set [theme variables](https://mermaid.js.org/config/theming.html#theme-variables), some of which sets inline styles on elements. We need to use `!important` to override the inline styles. We also had a lot of existing CSS, much of which already used `!important` to override mermaid CSS. Ideally we'd rewrite this all using theme variables in the future but this was a quick fix matching the current style for now

Reply:
Would this help?

https://mermaid.js.org/config/theming.html#customizing-themes-with-themevariables

Reply:
Yeah that's roughly what I linked but I mostly didn't want to migrate all the legacy CSS into that format